doc/user_guide/user-guide.md

```diff
@@ -12,49 +12,57 @@ GRANT CREATE SESSION TO api_service_account;
 GRANT SELECT ON SCHEMA my_schema TO api_service_account;
 ```
 
-### Configuration File
+### Properties
+
+#### Via YAML Configuration File
 
 The application needs a configuration file in yml format. The minimum required parameters are the credentials of the
-Exasol service account described above:
+Exasol service account described above and a list of API tokens (see [Authorization](#authorization)):
 
 ```yaml
-ExasolUser: "api_service_account"
-ExasolPassword: "secret_password"
+EXASOL_USER: "api_service_account"
+EXASOL_PASSWORD: "secret_password"
+API_TOKENS:
+  - "fwe3cqzE9pQblAYbLFRmxtN03uMgJ2"
```

Comment:
I somehow fear that someone copies it and forgets to change. Maybe add a comment or add -just-an-example- inside the tokens.

Reply:
Or different idea: Add a check in the implementation that blocks these three token and says that they are just examples and the user should use a custom one.

Reply:
I made them short in the example. If somebody copies it, it just won't work because this length is not allowed